src/i18n/en/translation.en.json

```diff
@@ -663,7 +663,7 @@
       },
       "sections": {
         "dashboard": {
-          "organization": "Facilitating organization",
+          "organization": "Leading organization",
```

Comment:
It's `hosting` for a hub

Reply:
@techsmyth

Reply:
On a hub it is "Facilitating organization", on a Challenge it is "Leading organizations". I just adjusted the translation file. This is good to go in...